# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xjdic23/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./xjdxgen.c_old	2003-02-04 17:32:52.000000000 +0100
+++ ./xjdxgen.c	2003-02-04 17:32:23.000000000 +0100
@@ -35,7 +35,7 @@
 #define TOKENLIM 40
 
 unsigned char *db;
-unsigned char ENVname[50];
+unsigned char ENVname[150];
 unsigned char *dicenv;
 struct stat *buf;
 unsigned long dbyte;
